### PR TITLE
Read aws s3 node

### PR DESCRIPTION
**Pull request by Jin1L**
### Description

- [x] I've included a concise description of what each node does

### Styleguide

- [x] My node adheres to the [styleguide](https://github.com/flojoy-io/nodes/blob/main/node_styleguide.md) for Flojoy nodes

### Docs

- [x] I've submitted a PR for a documentation page for the new node(s) that contains usage examples (see docs.flojoy.io)

### Testing

- [x] This PR includes a unit test ([example here](https://github.com/flojoy-io/nodes/blob/c28808f2d0aec8f116cdb3fccb46c4f5256574e9/TRANSFORMERS/ARITHMETIC/ADD/ADD_test_.py) and/or ideally a screenshot of the node's output on an example app.

### CODE REVIEW

**Comment by Jin1L** [comment]
S3_name refers to the name we use to refer to the access and secret access keys.
The table visualizer is a bit off. However, it displays the data that we retrieved from AWS S3 bucket

Screenshot 2023-06-19 at 1 42 35 PM
Screenshot 2023-06-19 at 1 45 00 PM

File: MANIFEST/read_s3.manifest.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+    parameters:
+      s3_name:
+        type: string
+        default: ""
+      bucket_name:
+        type: string
+        default: ""
+      file_name:
+        type: string
+        default: ""
```
Comment (scorinaldi) [inline MANIFEST/read_s3.manifest.yaml:14]: Can you add a newline at the end of this yaml please?

File: EXTRACTORS/FILE/READ_S3/READ_S3.py [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+@flojoy
+def READ_S3(dc_inputs: list[DataContainer], params: dict[str, str]):
+    """
+    The READ_S3 node takes S3_key name, S3 bucket name, and file name as input,
+    and extract the file from the specified bucket using the S3_key that they saved.
+
+    Parameters
+    ----------
+    s3_name:
```
Comment (scorinaldi) [inline EXTRACTORS/FILE/READ_S3/READ_S3.py:18]: Should you rename this to `s3_key_name` for clarity?

File: EXTRACTORS/FILE/READ_S3/READ_S3.py [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+        s3 = boto3.resource(
+            "s3", aws_access_key_id=s3_access_key, aws_secret_access_key=s3_secret_key
+        )
+        object = s3.Object(params["bucket_name"], params["file_name"])
+        buffer = io.BytesIO()
+        object.download_fileobj(buffer)
+        df = pd.read_parquet(buffer)
+
+        return DataContainer(type="dataframe", m=df)
+    except Exception as e:
```
Comment (scorinaldi) [inline EXTRACTORS/FILE/READ_S3/READ_S3.py:54]: Why include this try catch block at all if you just wind up re-raising whatever the Exception is?

**Review by scorinaldi — CHANGES_REQUESTED** [review]
Just a few questions and suggestions!

**Review by scorinaldi — APPROVED** [review]
LGTM

**Pull request merged**